pyani_plus/public_cli.py

```diff
@@ -588,7 +589,13 @@ def resume(  # noqa: C901, PLR0912, PLR0915
             binaries = {
                 "fastani": tool.exe_path,
             }
-            target_extension = ".fastani"
+            # Ideally this would be selective, as this will recompute every column
+            # (but we minimise this to a self-versus-self as fastANI won't no-op):
+            targets = [
+                f"all_vs_{Path(_.fasta_filename).stem}.fastani"
+                for _ in run.fasta_hashes
+            ]
```

Comment:
We could examine the full set of comparisons here to determine which subjects are already complete. i.e. which columns of the database do not need to be computed.

This would avoid the problematic behaviour of fastANI not liking being asked to do nothing vs a reference genome.